### PR TITLE
[Orangelight] Update orangelight ruby 3.1

### PR DESCRIPTION
**Pull request by christinach**
part of https://github.com/pulibrary/orangelight/pull/3294

### CODE REVIEW

**Review by kayiwa — APPROVED** [review]
Thanks! OK!

File: roles/orangelight/vars/main.yml [hunk 1]
```diff
@@ -7,5 +7,4 @@ rails_app_dependencies:
   - pkg-config
   - libtool
   - autoconf
-passenger_ruby: "/usr/bin/ruby2.7"
-ruby_version_override: "ruby2.7"
+
```
Comment (sandbergja) [inline roles/orangelight/vars/main.yml:10]: `molecule lint` is complaining about this extra line.  Could you please remove it?

**Review by sandbergja — CHANGES_REQUESTED** [review]

File: roles/orangelight/vars/main.yml (hunk 1 above)
Comment (christinach) [reply roles/orangelight/vars/main.yml:10]: yes I will.

**Comment by christinach** [comment]
catalog-staging is on ruby 3.1

**Review by sandbergja — APPROVED** [review]
Thanks, @christinach !

**Pull request merged**